### ISSUE TITLE
ModuleNotFoundError: No module named 'openlineage'

### ISSUE THREAD

**Issue by MatrixManAtYrService** [astronomer/astro-sdk#1256]
**Describe the bug**

`openlineage-airflow` is not listed as a dependency, yet we're throwing import errors if it's not present.  

Either it should be included as a dependency so that pip installs it (i.e. made a required dep) or we should handle cases where it isn't installed more gracefully.

This dockerfile encapsulates it:

```Dockerfile
FROM quay.io/astronomer/ap-airflow:2.4.2
RUN pip install astro-sdk-python
RUN python -c 'from astro.sql import load_file, run_raw_sql, transform'
```

Error:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/astro/.local/lib/python3.9/site-packages/astro/sql/__init__.py", line 5, in <module>
    from astro.sql.operators.append import AppendOperator, append
  File "/home/astro/.local/lib/python3.9/site-packages/astro/sql/operators/append.py", line 7, in <module>
    from openlineage.client.facet import (
ModuleNotFoundError: No module named 'openlineage'
```

**Version**
* Astro: astro-sdk-python==1.2.1
* OS: Debian

**To Reproduce**
`docker build .` in a directory with the dockerfile above

**Expected behavior**
No import errors.

**Additional context**

[To quote Ash
](https://astronomer.slack.com/archives/C02B8SPT93K/p1668545296014649?thread_ts=1668544708.307459&cid=C02B8SPT93K):
> Bug in astro-sdk, wasn't meant to be a hard dep
> It's an optional dep on astro-sdk, but the code mistakenly just imports it

**Comment by phanikumv** [comment]
Duplicate of https://github.com/astronomer/astro-sdk/issues/1243

**Comment by phanikumv** [comment]
Addressed in #1252
